source/package-manager/index.js

```diff
@@ -0,0 +1,65 @@
+import fs from 'node:fs';
+import path from 'node:path';
+import semver from 'semver';
+import {npmConfig, yarnBerryConfig, pnpmConfig, yarnConfig, npmConfigNoLockfile} from './configs.js';
```

Comment:
Thoughts on namespace importing these?

```suggestion
import * as configs from './configs.js';
```

If `np` supports other package managers in the future (e.g. if tools like Bun, Deno, or [Orogene](https://github.com/orogene/orogene) add publish commands), this would make adding them a little less verbose.